src/settingsView.script.ts

```diff
@@ -47,16 +47,63 @@ function updateProjects(projects: ProjectEntry[]) {
   unselectAllButton.hidden = !allEnabled;
 }
 
+function updateBrowsers(browsers: BrowserEntry[]) {
+  const browsersElement = document.getElementById('browsers') as HTMLUListElement;
+  browsersElement.innerHTML = '';
```

Comment:
I always thought that setting `textContent` is the most efficient way.

```suggestion
  browsersElement.textContent = '';
```